packages/storybook/src/stories/Checkbox.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Checkbox } from "@us-gov-cdc/cdc-react";
+import { ChangeEvent, useState } from "react";
+
+const meta: Meta<typeof Checkbox> = {
+  title: "Components/Checkbox",
+  tags: ["autodocs"],
+  component: Checkbox,
+  argTypes: {},
+};
+
+export default meta;
+type Story = StoryObj<typeof Checkbox>;
+
+export const Example: Story = {
+  args: {},
+  render: function Render() {
+    return (
+      <>
+        <Checkbox label="Single Checkbox" />
```

Comment:
Looks like I'm not able to toggle the checkbox for this story
![image](https://github.com/CDCgov/cdc-react/assets/98106992/37999540-6efa-45f2-80ee-5933a889c2ab)